src/gluonts/model/rotbaum/_predictor.py

```diff
@@ -282,7 +282,7 @@ def deserialize(
             return load_json(fp.read())
 
     def as_symbol_block_predictor(
-        self, batch: DataBatch
+        self, batch: DataBatch = None, dataset: Dataset = None
```

Comment:
If these are allowed to be `None`, then the proper annotation is `Optional[DataBatch]` and `Optional[Dataset]`

You may need to import `Optional` from `typing` for this

(Similarly elsewhere)

Reply:
Oh right! thanks!